.ci/setup_env_centos.sh

```diff
@@ -62,7 +62,7 @@ declare -A packages=( \
 	[bison_binary]="bison" \
 	[libgudev1-dev]="libgudev1-devel" \
 	[general_dependencies]="gpgme-devel glib2-devel glibc-devel bzip2 m4 gettext-devel automake alien autoconf pixman-devel coreutils" \
-	[build_tools]="python pkgconfig zlib-devel" \
+	[build_tools]="python3 pkgconfig zlib-devel" \
```

Comment:
does this work for the previous centos version as well, or are we abandoning that and moving on?

Reply:
not yet, yes it works on centos 7, we can trigger the CI of centos just to verify